src_c/display.c

```diff
@@ -2499,6 +2620,7 @@ pg_toggle_fullscreen(PyObject *self, PyObject *args)
                 }
                 display_surface->surf = SDL_GetWindowSurface(win);
                 pg_SetDefaultWindow(win);
+                PyErr_WarnEx(PyExc_Warning, "re-creating window in toggle_fullscreen" ,1);
```

Comment:
Missing check of return code from `PyErr_WarnEx`

Reply:
You were so right about the warnings. Obviously I didn't just *forget* to check the return code ten times or so, I had the completely wrong idea about how this works! In hindsight it *obviously* has to work this way, you can't just throw an exception in C code!